docs/_posts/2015-07-31-react-native-europe.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "ReactEurope Round-up"
+layout: post
+author: matthewjohnston4
+---
+
+A few weeks ago, the first React.js conference took place in the city of Paris, at ReactEurope. Attendees were treated to a range of talks covering React, React Native, Flux, Relay, and GraphQL. 
```

Comment:
"first React.js conference" sounds like it was the first ever. Could you qualify that? "first conference on European soil" or even just take their title "first React.js European conference"